src/raytracer/Material_library.txt

```diff
@@ -1,4 +1,4 @@
-"Primary key","Reflector","mu (k & precursor)","sigma (k & precursor)","mu (k & postcursor)","sigma (k & precursor)","mu (Y & precursor)","sigma (Y & precursor)","mu (Y & postcursor)","sigma (Y & precursor)","mu (lambda & precursor)","sigma (lambda & precursor)","mu (lambda & postcursor)","sigma (lambda & precursor)","mu (sigma theta)","sigma (sigma theta)","mu (RL)","sigma (RL)","Dielectric constant"
+"Primary key","Reflector","mu (k & precursor)","sigma (k & precursor)","mu (k & postcursor)","sigma (k & postcursor)","mu (Y & precursor)","sigma (Y & precursor)","mu (Y & postcursor)","sigma (Y & postcursor)","mu (lambda & precursor)","sigma (lambda & precursor)","mu (lambda & postcursor)","sigma (lambda & postcursor)","mu (sigma theta)","sigma (sigma theta)","mu (RL)","sigma (RL)","Dielectric constant"
```

Comment:
is this library still used? if not, remove it

Reply:
This is redundant. I forgot to delete it.